### PR TITLE
Added shell script to install python dependencies

### PR DESCRIPTION
**Pull request by bsrinivasa**
Python dependencies required during setup of the tool for the first time. 

cc: @amishas157

### CODE REVIEW

**Comment by bsrinivasa** [comment]
Added shell script to setup python dev version on linux machine if the python version is not higher than 2.6

File: setup.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#python dependencies required for the tool
```
Comment (amishas157) [inline setup.sh]: We can make a requirements file for this. For example: https://github.com/mapbox/wikimama/blob/master/requirements.txt

**Review by amishas157 — APPROVED** [review]

**Comment by amishas157** [comment]
@bsrinivasa Thanks for the PR, It looks all good to me. Let's merge after getting the requirements file and adding docs for using these scripts in README.md ?

**Comment by bsrinivasa** [comment]
@amishas157 have updated the python package dependencies in a text file. Here are the new changes.
- Added python dependencies with appropriate version numbers to `requirements.txt`
- Updated README.md with the instructions supporting ^.

**Comment by amishas157** [comment]
Yay, Awesome. Thanks

**Pull request merged**